Update README : Usage Section (#21)
cod-ed/handwrite: README.md

    </a>
</p>

# Handwrite - Type in your Handwriting!

Ever had those long-winded assignments, that the teacher always wants handwritten?
Is your written work messy, cos you think faster than you can write?
Now, you can finish them with the ease of typing in your own font!

> ![handwrite](https://raw.githubusercontent.com/cod-ed/assets/handwrite/handwrite.gif)
> Handwrite makes typing written assignments efficient, convenient and authentic.

Reviewer: ```suggestion
![handwrite](https://raw.githubusercontent.com/cod-ed/assets/handwrite/handwrite.gif)

Handwrite makes typing written assignments efficient, convenient and authentic.
```

`python setup.py test`

You are ready to go!

# Usage

## Creating your Handwritten Sample 

> 1. Take a printout of `handwrite_sample.pdf`, present in the cloned `handwrite` folder.

Reviewer: Since this will be package, not everybody will clone the project. Some might pip install. 

Better would be to just paste the link to file.
```suggestion
1. Take a printout of the [sample form](https://github.com/cod-ed/handwrite/raw/main/handwrite_sample.pdf).
```

> 2. Fill the form. (Take a look at `handwrite/temp_sheets/sheet_new.jpg` as an example)

Reviewer: Link to something in assets coz we'll get rid of these files soon.

3. Scan the filled form using a scanner, or Adobe Scan in your phone.

> 4. Save the `.jpg` image in your system.

Reviewer: Just a preference thing, can we leave blank line after each point, looks a little neat.

> 4. Add the jpg image to the `temp_sheets` folder.

Reviewer: The image can be anywhere, usage: `handwrite PATHTOIMAGE OUTPUT_NAME`

Your form should look like this:

> ![Filled form](temp_sheets/sheet_new.jpg)

Reviewer: Link to Assets

> ![Filled form](https://raw.githubusercontent.com/cod-ed/assets/handwrite/handwrite_filled_form.jpg)

Reviewer: Can generate a font with this image and add a ss or gif after the Creating your font section? Will look nice.

## Creating your font

> 1. Activate your environment and go to the cloned folder.

Reviewer: Neither environment needed nor cloning. Make sure handwrite, potrace and fontforge are installed.

2. `handwrite -h` shows you all the arguments you can use.

> 3. To create your font, type: `handwrite temp_sheets/your_font.jpg your_font_name.ttf`

Reviewer: Copy this from help maybe so it is consistent everywhere and 
```suggestion
3. To create your font, type:
`handwrite temp_sheets/your_font.jpg your_font_name.ttf`
```

> 4. Your font will be created as `handwrite/your_font_name.ttf`. Open it, and select install.

Reviewer: Again wherever the command will be called

> 4. Select your font in your word processor and get to work!

Reviewer: same here
